Util/AdminObjectAclManipulator.php

```diff
@@ -272,7 +272,7 @@ protected function buildForm(AdminObjectAclData $data, FormBuilderInterface $for
                 );
             }
 
-            $formBuilder->add($key, new AclMatrixType(), array('permissions' => $permissions, 'acl_value' => $aclValue));
+            $formBuilder->add($key, AclMatrixType::class, array('permissions' => $permissions, 'acl_value' => $aclValue));
```

Comment:
I am the last person who would support these old crap... But we use semver, so we must support it until the next major release.